integration/example-library/meltano-expand-envvars-in-array/index.md

```diff
@@ -0,0 +1,19 @@
+# Expanding environment variables in elements of an array setting value
+
+This is a simple example of how to expand environment variables in elements of an array setting value.
+
+## Setup
+
+Install the project dependencies:
+
+```shell
+meltano install
+```
+
+## Run the pipeline
+
+```shell
+ANIMAL_GROUP=birds meltano invoke tap-smoke-test
+ANIMAL_GROUP=reptiles meltano invoke tap-smoke-test
+ANIMAL_GROUP=mammals meltano invoke tap-smoke-test
```

Comment:
These expand this config

```yaml
- stream_name: $ANIMAL_GROUP
  input_filename: $MELTANO_PROJECT_ROOT/data/$ANIMAL_GROUP.jsonl
```

into different values and thus different pipelines :)